generator/src/pkg/service/util.go

```diff
@@ -73,14 +82,17 @@ func CreateDeployment(metadataName, selectorAppName, selectorClusterName string,
 func CreateDeploymentWithAffinity(metadataName, selectorAppName, selectorClusterName string, numberOfReplicas int,
```

Comment:
The functions CreateDeployment() and CreateDeploymentWithAffinity() share several lines of similar code... we may consider here how to avoid duplicated code.